static/src/javascripts/projects/common/modules/atoms/youtube-player.js

```diff
@@ -48,12 +51,14 @@ interface AdsConfig {
 
 let tcfState = null;
 let ccpaState = null;
+let tcfData = {};
 onConsentChange(state => {
     if (state.ccpa) {
         ccpaState = state.doNotSell;
     } else {
-        tcfState = state.tcfv2
-            ? Object.values(state.tcfv2.consents).every(Boolean)
+        tcfData = state.tcfv2;
+        tcfState = tcfData
+            ? Object.values(tcfData.consents).every(Boolean)
             : state[1] && state[2] && state[3] && state[4] && state[5];
```

Comment:
This looks like TCFv1 stuff…

Reply:
good catch. there were a lot of tcfv1 stuff lying around, i removed those that were relying on sourcepoint id of custom vendor when i did the refactor for using the `getConsentFor`.
I also think we should use the a custom sourcepoint id for this vendor and use the `getConsentFor` instead of relying on having all consents on. will check with Alex A for subsequent PR. for now just removed the tcfv1 stuff

Reply:
checked with Alex, we are good without SP id, so no subsequent PR needed :)